frontend/src/components/FormElements/formelements.module.css

```diff
@@ -16,7 +16,7 @@
 
 .timeInput,
 .dateInput {
-  font-size: 1.25rem;
+  font-size: 1rem;
```

Comment:
Changing the font size here makes the text in the add new ride modal seem weirdly small. It may be better to change this text in the add rider CSS module instead of changing it here